cluster/gce/util.sh

```diff
@@ -53,6 +53,17 @@ function join_csv {
 
 # Verify prereqs
 function verify-prereqs {
+  if [[ "${ENABLE_EXPERIMENTAL_API}" == "true" ]]; then
```

Comment:
Is this going to work if someone specifies ```RUNTIME_CONFIG``` manually? I think we should just append our part to it. WDYT?